lights-out.js

```diff
@@ -16,4 +17,51 @@ class LightsOut {
             }
         }
     }
+
+    getNeighbors(row, col) {
+        const neighborArray = [];
+        
+        if(row < this.numRows - 1) {
+            if(row > 0) {
+                neighborArray.push([row + 1, col]);
+                neighborArray.push([row - 1, col]);
+            } else {
+                neighborArray.push([row + 1, col]);
+                neighborArray.push([this.numRows - 1, col]);
+            }
+        } else {
+            neighborArray.push([0, col]);
+            neighborArray.push([row - 1, col]);
+        }
+        if(col < this.numCols - 1) {
+            if(col > 0) {
+                neighborArray.push([row, col + 1]);

```

Comment:
Similar to my previous comment, the line `neighborArray.push([row, col + 1]);` appears twice. Delete those lines please, and move the line to above the `if (col >0)` conditional

Reply:
Moved `neighborArray.push([row, col + 1]);` to above the `if (col >0)` conditional.